pages/components/docs/Box.md

```diff
@@ -5,7 +5,7 @@ The Box component serves as a wrapper component for most layout related needs. U
 ## Default example
 
 ```.jsx
-<Box>
+<Box display='inline-block'>
```

Comment:
This shouldn't work because `display` isn't included in `LAYOUT`, but it still does